yaksh/views.py

```diff
@@ -731,9 +814,12 @@ def get_result(request, uid, course_id, module_id):
         next_question, error_message, paper = _update_paper(request, uid,
                                                             result
                                                             )
+        answer = Answer.objects.get(id=uid)
+        current_question = answer.question
```

Comment:
Food for thought, if I have a question with time still left, and I give the correct answer, shouldn't I be able to go to the next question?

Reply:
Considering that we have solution in the video so the user should wait till the question time is over.